### ISSUE TITLE
sound distortion

### ISSUE THREAD

**Issue by 909oce** [PeytonPlayz595/Alpha-v1.2.6#3]
audio sounds super bitcrushed when playing both your release and hosting myself. unsure if this is something on my end or your end.

**Comment by PeytonPlayz595** [comment]
> thats just compression. it would be nice if there was an uncompressed version i did take some sounds from the original game to convert to mp3 and replace but i coulnt find every sound

Yeah, I compressed all the audio files and converted them to mp3's to try and make the file size smaller, I didn't do this in Beta 1.7.3 for this exact reason, I left them uncompressed and as ogg file's.
